docs/ch1p1/[2]automatic-gradient.mdx

```diff
@@ -1,5 +1,11 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # 自动求梯度
 
+<Tabs groupId="language">
+  <TabItem value="tf" default>
+      
```

Comment:
```suggestion

```

It's broken because there are space chars in the "empty" line. It is not really empty.

Reply:
I deleted it. But the broken is still there.

Reply:
![image](https://user-images.githubusercontent.com/55493212/141940608-6a5074de-11cf-4647-8659-83003a353631.png)

Reply:
fixed it

Reply:
Weird. The new commit deleted the whole line.